tests/checkBlockTranslationCoverage.js

```
@@ -77,86 +23,10 @@ async function test() {
 			}
 			blockNames.push(blockName);
 		});
-
+		
 		let materialList = await new MaterialList(blockMetadata, itemMetadata, translationFile);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'materialList' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)